worldedit-bukkit/adapters/adapter-1_21_6/src/main/java/com/sk89q/worldedit/bukkit/adapter/ext/fawe/v1_21_6/PaperweightAdapter.java

```diff
@@ -204,6 +204,14 @@ public final class PaperweightAdapter implements BukkitImplAdapter<net.minecraft
 
     private static final RandomSource random = RandomSource.create();
 
+    private static final String WRONG_VERSION =
+            """
+            This version of WorldEdit has not been tested with the current Minecraft version.
+            While it may work, there might be unexpected issues.
+            It is recommended to use a version of WorldEdit that supports your Minecraft version.
+            For more information, see https://worldedit.enginehub.org/en/latest/faq/#bukkit-adapters
```

Comment:
WorldEdit -> FastAsyncWorldEdit

And I don't think we link to WorldEdit docs - so that line can be removed ig